storm-core/src/jvm/backtype/storm/serialization/Test.java

```diff
@@ -0,0 +1,17 @@
+package backtype.storm.serialization;
```

Comment:
This file appears to have been added by mistake, if not it should be turned into a true unit test, with apache header etc.